src/main/java/frc/robot/BallChucker9000.java

```diff
@@ -76,7 +76,7 @@ public void update() {
      * @param power positive is counter clockwise
      */
     public void moveRotator(double power) {
-        rotatorMotor.set(power);
+        rotatorMotor.set(0.1 * power);
```

Comment:
For best limiting of the power, we should put limit the input values to [-1, 1]. Otherwise, an enterprising individual could `moveRotator(10);`.